.github/actions/export-ocm-fragments/action.yaml

```diff
@@ -0,0 +1,175 @@
+    description: |
+      a YAML document, or a sequence of multiple YAML documents containing the OCM-Resources to
+      export. Each YAML document may either be mapping (in which case it is interpreted as a single
+      resource), or a list of mappings (in which case each list-entry is interpreted as a single
+      resource).
+  ocm-resources-file:
+    required: false
+    type: string
+    description: |
+      a path to a YAML-file containing OCM-Resources (interpreted the same way as `ocm-resources`
+      input).
+  ocm-sources:
+    required: false
+    type: string
+    description: |
+      a YAML document, or a sequence of multiple YAML documents containing the OCM-Sources to
+      export. Each YAML document may either be mapping (in which case it is interpreted as a single
+      Source), or a list of mappings (in which case each list-entry is interpreted as a single
+      Source).
+  ocm-sources-file:
+    required: false
+    type: string
+    description: |
+      a path to a YAML-file containing OCM-Resources (interpreted the same way as `ocm-sources`
+      input).
+  blobs-directory:
+    required: false
+    default: blobs.d
+    type: string
+    description: |
+      path to a directory containing local-blobs referenced by any of the passed ocm-artefacts.
+      blobfiles are expected to be regular files residing immediately below this directory, with
+      a naming-schema matching local-blob-ref (`<algorithm>:<hexdigest>`). For example:
+      `blobs.d/sha256:01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b`
+
+  ctx:
+    required: false
+    type: string
+    description: |
+      an optional ctx that will be added into artefact-name, and filenames. Useful if multiple
+      component-descriptors are to be created in the same workflow.
+
+outputs:
+  artefact-name:
+    description: |
+      the name of the uploaded artefact
+    value: ${{ steps.merge.outputs.artefact-name }}
+
+  tarfile-name:
+    description: |
+      the name of the tarfile contained in the uploaded artefact
+    value: ${{ steps.merge.outputs.artefact-tarfile }}
+
+
+runs:
+  using: composite
+  steps:
+    - name: install-gardener-gha-libs
+      uses: gardener/cc-utils/.github/actions/install-gardener-gha-libs@master
+    - name: merge-ocm-fragments
+      id: merge
+      shell: python
+      run: |
+        import os
+        import sys
+        sys.path.insert(1, os.environ['GITHUB_ACTION_PATH'])
+
+        import hashlib
+        import io
+        import tarfile
+        import tempfile
+        import pprint
+
+        import yaml
+
+        import export_ocm
+
+        resources = list(
+          export_ocm.iter_artefacts(
+            artefacts_yaml='''${{ inputs.ocm-resources }}''',
+            artefacts_file='${{ inputs.ocm-resources-file }}',
+          ),
+        )
+        sources = list(
+          export_ocm.iter_artefacts(
+            artefacts_yaml='''
+              ${{ inputs.ocm-sources }}
```

Comment:
please keep this consistent with resource (`artefacts_yaml='''${{ inputs.ocm-resources }}'''`)

Reply:
that's odd. I also changed that. so it seems I did not push the correct commit(s)